docs/source/markdown/podman-artifact-add.1.md.in

```diff
@@ -31,6 +31,11 @@ Set the media type of the artifact file instead of allowing detection to determi
 
 Print usage statement.
 
+#### **--replace**
```

Comment:
This is duplicated, you can use `@@option replace` and do a small edit in `options/replace.md` to add artifacts in there